### DISCUSSION TITLE
Kbar shrotcuts and theming

### DISCUSSION THREAD

**Discussion by grigoreo-fox** [refinedev/refine#5750] [Ideas]
I propose adding a shortcuts field to the resources configuration to enable keyboard shortcuts in kbar for common actions (list, create, edit) directly within the Refine framework. Here's an example configuration:

```ts
resources={[
  {
    name: "posts",
    list: "/posts",
    create: "/posts/create",
    edit: "/posts/edit/:id",
    meta: {
      shortcuts: {
        list: ["p"],
        create: ["c", "p"],
        edit: ["e"],
      },
    },
  },
]}
```

Additionally, I'm interested in learning about straightforward methods to customize the Kbar theme within Refine.

**Comment by BatuhanW** [comment]
Hey @grigoreo-fox looks like a good idea. We are open to contributions from the community, if anyone wants to work on this.

**Reply by grigoreo-fox** [reply]
Please, check this out #5782
